linode/manager
# upcoming: [M3-7300] - Replace Linode details Analytics tab with Recharts

## Description 📝
Replace Linode details Analytics charts with Recharts

- CPU % chart
- Disk I/O chart
- Network Transfer charts (IPv4 & IPv6)

## Changes  🔄
List any change relevant to the reviewer.
- Updated UI charts for the Linode details Analytics tab
- Added custom legend logic to AreaChart
- Migrate MetricDisplay styles to styled components

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![image](https://github.com/linode/manager/assets/115299789/5e693603-afde-4945-9cf8-36fcd497e73a) | ![image](https://github.com/linode/manager/assets/115299789/ead68364-91c3-44d9-b59e-8d22b4f75f76) |

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Have a Linode that's been running for a while

### Verification steps 
(How to verify changes)
- Go to a Linode's details page. On the Analytics tab, verify the updated UI and that there are no regressions in the graph, units, functionality, etc.
  - Note: X and Y axis labels will not match exactly but the graph data is still the same

## As an Author I have considered 🤔

*Check all that apply*
